lib/redix/utils.ex

```diff
@@ -90,11 +90,17 @@ defmodule Redix.Utils do
     Connection.reply(from, {request_id, reply})
   end
 
+  defp check_port!(port) when is_integer(port), do: port
+  defp check_port!(port) do
+    raise ArgumentError, "invalid port number type: #{inspect port}." <>
+      " The port number expects an integer value."
```

Comment:
Let's make the error message `"expected an integer as the value of the :port option, got: #{inspect(port)}"` :)